### DISCUSSION TITLE
Joke: are those commit messages authorized?

### DISCUSSION THREAD

**Discussion by grinlemon** [heig-vd-dai-course/heig-vd-dai-course#76] [Q&A]
<img width="362" alt="notfunnycommits" src="https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/73121793/0752a92b-1c34-4241-b4f0-d305b6726ba6">

**Comment by ludelafo** [comment]
It depends! Did you really create Batman in the commit you did? Shall I call you Dr. Frankenstein? 😄
